sematic/resolvers/silent_resolver.py

```diff
@@ -24,6 +24,7 @@ def _run_inline(self, future: AbstractFuture) -> None:
             self._update_future_with_value(future, value)
         except Exception as exception:
             self._handle_future_failure(future, exception)
+            return
```

Comment:
This was not necessary until now because `_handle...` would always raise. Now it only raises if no retry is attempted.

Reply:
What does the `return` actually do? The finally block still executes even if you `return` from within the try (which we do want--calling `_end_inline_execution` allows us to identify the end of the logs for the inline execution)

Reply:
Ah right, so I should move `return` to the `finally` block actually.